message_app/static/js/images/main.js

```diff
@@ -0,0 +1,55 @@
+// ===== Images =====
+class ImageModel {
+    constructor(conversationId, sender_name, thumbnail_source, regular_source, width, height, seen, timestamp) {
+        this.conversationId = conversationId;
+        this.sender_name = sender_name;
+        this.thumbnail_source = thumbnail_source;
+        this.regular_source = regular_source;
+        this.width = width;
+        this.height = height;
+        this.seen = seen; // TODO: Implement seen function
```

Comment:
Can `ImageModel `be a subclass of `MessageModel `?

Reply:
It is actually can be a subclass. I will do some modification in the next commit.